<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErrorDlg</class>
 <widget class="QDialog" name="ErrorDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>71</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Error</string>
  </property>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
